account/state/state.go

```diff
@@ -33,7 +33,7 @@ type StorageGetter interface {
 }
 
 type StorageSetter interface {
-	// Store a 32-byte value at key for the account at address
+	// Store a 32-byte value at key for the account at address, setting to Zero256 removes the key
```

Comment:
A comment along these lines exists in other places - I thought here too but apparently not. Adding it since it should be remarked.